.github/workflows/push.yml

```
@@ -497,71 +535,87 @@
         id: gather
         run: |
           set -euo pipefail
-          AFFECTED_PROJECTS="$(echo '${{ matrix.chunk }}' | jq -r '.projects')"
+          AFFECTED_PROJECTS="$(echo '${{ matrix.chunk }}' | jq -r '.project')"
           DOCKER_TYPE="$(echo '${{ matrix.chunk }}' | jq -r '.docker_type')"
+          APP_HOME="$(echo '${{ matrix.chunk }}' | jq -r '.home')"
           echo AFFECTED_PROJECTS="$AFFECTED_PROJECTS" >> "$GITHUB_ENV"
           echo DOCKER_TYPE="$DOCKER_TYPE" >> "$GITHUB_ENV"
+          echo APP_HOME="$APP_HOME" >> "$GITHUB_ENV"
         continue-on-error: true
       - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)